clients/tfchain-client-go/contract.go

```diff
@@ -233,6 +233,113 @@ func (s *Substrate) CreateNodeContract(identity Identity, node uint32, body stri
 	return s.GetContractWithHash(node, h)
 }
 
+// BatchCreateContractData struct for batch create contract
+type BatchCreateContractData struct {
+	Node               uint32
+	Body               string
+	Hash               string
+	PublicIPs          uint32
+	SolutionProviderID *uint64
+
+	// for name contracts. if set the contract is assumed to be a name contract
+	// and other fields are ignored
+	Name string
+
+	hexHash HexHash
```

Comment:
This field will be private, it doesn't start with upper case.

Reply:
that was intentional as it is generated internally and not provided by user, should it be public?